src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -2744,6 +3186,7 @@ private static void UpdateIndexForBackup(DatabaseRecord databaseRecord, string t
                 case nameof(EditTimeSeriesConfigurationCommand):
                 case nameof(PutAutoIndexCommand):
                 case nameof(PutDatabaseClientConfigurationCommand):
+                case nameof(EditDatabaseClientConfigurationCommand):
```

Comment:
this was ordered alphabetically for easier sync :)

Reply:
it's easier to search commands when they are near each other :)
reverted.

Reply:
Unless after some time half of the commands that should be here is missing and you need to figure out which ones are those :)